toonz/sources/toonzqt/fxschematicnode.cpp

```diff
@@ -1239,22 +1239,20 @@ void FxSchematicPort::paint(QPainter *painter,
                             const QStyleOptionGraphicsItem *option,
                             QWidget *widget) {
   if (m_isPassThrough && getLinkCount() > 0) return;
-
+  int devicePixelRatio = getDevicePixelRatio(widget);
```

Comment:
@shun-iwasawa 

FYI, when compiled on QT < 5.14, this line leads to a crash further down when painting the Fx Schematic's Output parent port.  Crash only occurs when the Fx Schematic is shown for the 1st time on the second low res monitor but the main monitor is high res.

As you are not using devicePixelRatio anywhere in this procedure, I verified it can be safely removed.